packages/next/build/webpack/loaders/postcss-loader/src/Warning.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * **PostCSS Plugin Warning**
+ *
+ * Loader wrapper for postcss plugin warnings (`root.messages`)
+ *
+ * @class Warning
+ * @extends Error
+ *
+ * @param {Object} warning PostCSS Warning
+ */
+class Warning extends Error {
+  constructor(warning) {
+    super(warning)
+
+    const { text, line, column, plugin } = warning
+
+    this.name = 'Warning'
+
+    this.message = `${this.name}\n\n`
+
+    if (typeof line !== 'undefined') {
+      this.message += `(${line}:${column}) `
+    }
+
+    this.message += plugin ? `${plugin}: ` : ''
+    this.message += `${text}`
+
+    this.stack = false
+  }
+}
+
+module.exports = Warning
```

Comment:
This looks nearly identical to `loaders/css-loader/src/Warning.js`. Could they both share the same file?

Reply:
I think we can do that in a follow up, this PR is just to make sure that postcss-loader/css-loader live in Next.js

Reply:
Opened a PR here: https://github.com/vercel/next.js/pull/28727